libgalois/include/katana/GraphTopology.h

```diff
@@ -1032,6 +1202,7 @@ using PGViewNodesSortedByDegreeEdgesSortedByDestID =
 using PGViewBiDirectional = BasicPropGraphViewWrapper<SimpleBiDirTopology>;
 using PGViewEdgeTypeAwareBiDir =
     BasicPropGraphViewWrapper<EdgeTypeAwareBiDirTopology>;
+using PGViewProjectedGraph = BasicPropGraphViewWrapper<ProjectedTopology>;
```

Comment:
@uditagar : you'll need to introduce a `ProjectedTopologyWrapper` which can be simply aliased to `BasicTopologyWrapper<ProjectedTopology>`. Reason being, `BasicPropGraphViewWrapper` takes topology wrappers by copy and `ProjectedTopology` cannot (and should not) be copied. `BasicTopologyWrapper` takes a topology object type by pointer and can be copied trivially.

Reply:
I get what you are saying. So should I need to do this `using PGViewProjectedGraph = BasicTopologyWrapper<ProjectedTopology>` or do I need to create a new class. And if so, what should be the contents of this new class? Thanks

Reply:
added new wrapper now